.packit.yaml

```diff
@@ -13,8 +13,8 @@ jobs:
   trigger: pull_request
   metadata:
     targets:
-    - fedora-30-x86_64
-    - fedora-31-x86_64
+    - fedora-32-x86_64
+    - fedora-33-x86_64
```

Comment:
Why not 'fedora-stable'?

Reply:
No idea why it's explicitly enumerated, I thought it's for testing because there are `fedora-stable` and `fedora-all` used in other jobs.

Reply:
Because we didn't change master for a long time. All testing PRs now have aliases and there is one for hardcoded ones (#129).

We can use what we need/want in master.

Reply:
@lachmanfrantisek will you merge #129?

Reply:
No, it's used for the testing purposes to have at least one test case with targets specified.

Reply:
Ok, I didn't realize #129 is for validation. So can I merge this one as is and you'd update #129?
Or what would you suggest? (to avoid https://github.com/packit/hello-world/pull/150#issuecomment-691929903 which made me create this PR)

Reply:
What about using `fedora-stable` alias (in this PR) so we don't need to update anthing in the future. I'll correct the test PRs if needed when merged.

> Ok, I didn't realize #129 is for validation. 

The script filters all PRs starting with `Basic test case: `